lilypond/songs/silent-night-holy-night/silent-night-holy-night.ly

```diff
@@ -0,0 +1,94 @@
+\version "2.22.1"
+#(ly:set-option 'relative-includes #t)
+\include "../../lib/hymn-common.ily"
+
+%% TUNE INFO
+composer = \smallText "Music: Franz Gruber, 1818"
+meter = \smallText "Irregular"
```

Comment:
You had no way of knowing this, but the web pages won't generate without a tune name in the meter (sometimes it's the same as the title). For this, based on hymnary.org, I suggest:

`meter = \smallText "STILLE NACHT irregular"`

Reply:
added!